index.html

```diff
@@ -38,13 +30,26 @@
                 <li>Contact</li>
                 <li><i class="bi bi-envelope emailsend"></i></li>
             </ul>
-            <i class="bi bi-list menu"></i>
+            <i id="Deploy_menu" class="bi bi-list menu"></i>
+            <ul id="mobile-menu">
+                <li id="close-btn" class="close-mobile-menu "><i class="bi bi-x-lg"></i></li>
+                <li class="line_border">
+                    <a href="#proyects">Portofolio</a>
+                </li>
+                <li class="line_border">
+                    <a href="#aboutme">About Me</a>
+                </li>
+                <li class="line_border">
+                    <a href="#contact">
+                        Contact Me
+                    </a>
+                </li>
+            </ul>
```

Comment:
- [ ] Your mobile menu works fine, however, a few adjustments is needed with the positioning of the hamburger and the close button. Your hamburger menu and the close button are not in the same position when switched. Following the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471), let's ensure that the hamburger and the close button are on the same position, and doing so will enhance great user experience. 

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/188524934-f2505f00-cfe3-4f21-b92e-b763b1a9e473.png)


**Figma design:**
![image](https://user-images.githubusercontent.com/12745474/188524807-0cbd96f4-42c9-478d-b5ff-a970557053f6.png)

Reply:
Ok im going to fix that